Tests/SwiftyJSONTests/PrintableTests.swift

```diff
@@ -120,5 +120,28 @@ class PrintableTests: XCTestCase {
         XCTAssertTrue(description.range(of: "\"1\":2", options: NSString.CompareOptions.caseInsensitive) != nil)
         XCTAssertTrue(description.range(of: "\"2\":\"two\"", options: NSString.CompareOptions.caseInsensitive) != nil)
         XCTAssertTrue(description.range(of: "\"3\":null", options: NSString.CompareOptions.caseInsensitive) != nil)
-    }
+	}
+	
+	func testJson() {
+		let dictionary = ["1":2,"2":2,"three":3,"list":["aa","bb","dd"], "foo": []] as NSDictionary
```

Comment:
nitpick: this dictionary should format with a trailing space after `:`, just like line 135.